src/LICENSE

```diff
@@ -0,0 +1,19 @@
+The MIT License (MIT)
+Copyright (c) <year> <copyright holders>
```

Comment:
@GuiltyDolphin you're right, it's better to fill these blanks. I committed a change where (c) holders are credited similarly with another part of community-platform. Please do tell if it's reasonable enough or something else should be mentioned.